<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adddata</class>
 <widget class="QWidget" name="adddata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   border-radius: 10px;
   border: 3px groove blue;
   background-color:rgba(32, 25, 255,0.8);;
   color:rgba(255, 255, 255,0.5);
   
}
QPushButton:hover{
   border-radius: 10px;
   border: 3px groove white;
   background-color:rgba(32, 25, 255,1);;
   color:rgb(255, 255, 255);
}
QPushButton:pressed{
   border-radius: 10px;
   border: 3px groove black;
   background-color:rgba(110, 209, 255,0.9);;
   color:rgba(255, 255, 255,0.9);
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   border-radius: 10px;
   border: 3px groove blue;
   background-color:rgba(32, 25, 255,0.8);;
   color:rgba(255, 255, 255,0.5);
   
}
QPushButton:hover{
   border-radius: 10px;
   border: 3px groove white;
   background-color:rgba(32, 25, 255,1);;
   color:rgb(255, 255, 255);
}
QPushButton:pressed{
   border-radius: 10px;
   border: 3px groove black;
   background-color:rgba(110, 209, 255,0.9);;
   color:rgba(255, 255, 255,0.9);
}</string>
   </property>
   <property name="text">
    <string>确认增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   border-radius: 10px;
   border: 3px groove blue;
   background-color:rgba(32, 25, 255,0.8);;
   color:rgba(255, 255, 255,0.5);
   
}
QPushButton:hover{
   border-radius: 10px;
   border: 3px groove white;
   background-color:rgba(32, 25, 255,1);;
   color:rgb(255, 255, 255);
}
QPushButton:pressed{
   border-radius: 10px;
   border: 3px groove black;
   background-color:rgba(110, 209, 255,0.9);;
   color:rgba(255, 255, 255,0.9);
}</string>
   </property>
   <property name="text">
    <string>确认删除</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>462</width>
     <height>268</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>汉仪秦川飞影W</family>
              <pointsize>15</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
            </property>
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>汉仪秦川飞影W</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
            </property>
            <property name="text">
             <string>性别</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>汉仪秦川飞影W</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
            </property>
            <property name="text">
             <string>工号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
          </property>
          <property name="text">
           <string>联系电话</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
          </property>
          <property name="text">
           <string>系</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
          </property>
          <property name="text">
           <string>学历</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
          </property>
          <property name="text">
           <string>学院</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="namelineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="sexlineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="numlineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="tellineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="deplineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="deglineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="collegelineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>汉仪秦川飞影W</family>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    color:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>删除工号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="delnumlineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>-10</y>
     <width>811</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
    background-image:url(:/new/1/others/2.jpg)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255))
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>120</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255))
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>returnButton</zorder>
  <zorder>addButton</zorder>
  <zorder>delButton</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
